002_parsers.md

```diff
@@ -70,7 +71,7 @@ it over the result of second parse function. Since the parser operation yields a
 list of tuples, composing a second parser function simply maps itself over the
 resulting list and concat's the resulting nested list of lists into a single
 flat list in the usual list monad fashion. The unit operation injects a single
-pure value into the parse stream.
+pure value as result without reading from the parse stream.
```

Comment:
Yes, that's fine.
Original formulation sounded like we're manipulating the input.